web/modules/custom/dpl_opening_hours/src/Controller/OpeningHoursEditorController.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Drupal\dpl_opening_hours\Controller;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\node\NodeInterface;
+use Drupal\taxonomy\TermInterface;
+
+/**
+ * Defines OpeningHoursEditorController class.
+ */
+class OpeningHoursEditorController extends ControllerBase {
+
+  /**
+   * Display the opening hours app.
+   *
+   * @param int $node
+   *   The node ID.
+   *
+   * @return mixed[]
+   *   The app render array.
+   */
+  public function content(int $node): array {
+    // Load the node entity.
+    $nodeStorage = $this->entityTypeManager()->getStorage('node');
+    $nodeEntity = $nodeStorage->load($node);
+
+    if (!$nodeEntity instanceof NodeInterface || $nodeEntity->getType() !== 'branch') {
+      return [];
+    }
+
+    // Load taxonomy terms for opening_hours_categories.
+    $taxonomyStorage = $this->entityTypeManager()->getStorage('taxonomy_term');
+    $openingHoursCategories = $taxonomyStorage->loadByProperties([
+      "vid" => "opening_hours_categories",
+    ]);
```

Comment:
Please reconsider the commenting here. 

It is not very helpful. It simply states what pretty readable code is doing.